properties_pane/entity_level/entityLevelConfig.json

```diff
@@ -154,6 +154,83 @@ making sure that you maintain a proper JSON format.
 					"value": "Vault"
 				}
 			},
+			{
+				"propertyName": "As Dynamic",
+				"propertyKeyword": "dynamic",
+				"propertyType": "checkbox"
+			},
+			{
+				"propertyName": "Iceberg",
+				"propertyKeyword": "iceberg",
+				"propertyType": "checkbox",
+				"dependency": {
+					"key": "dynamic",
+					"value": true
+				}
+			},
+			{
+				"propertyName": "External volume",
+				"propertyKeyword": "externalVolume",
+				"propertyType": "text",
+				"dependency": {
+					"key": "iceberg",
+					"value": true
+				}
+			},
+			{
+				"propertyName": "Catalog",
+				"propertyKeyword": "catalog",
+				"propertyType": "text",
+				"dependency": {
+					"key": "iceberg",
+					"value": true
+				}
+			},
+			{
+				"propertyName": "Base location",
+				"propertyKeyword": "baseLocation",
+				"propertyType": "text",
+				"dependency": {
+					"key": "iceberg",
+					"value": true
+				}
+			},
+			{
+				"propertyName": "Target Lag",
+				"propertyKeyword": "targetLag",
+				"propertyType": "group",
```

Comment:
Seems, there is only one `TARGET_LAG` option available, better change the propertyType to `block`